packages/contracts-bedrock/scripts/DeployPeripheryConfig.s.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Script } from "forge-std/Script.sol";
+import { console2 as console } from "forge-std/console2.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+
+/// @title DeployPeripheryConfig
+/// @notice Represents the configuration required to deploy the periphery contracts. It is expected
```

Comment:
I think this reads a bit strange and would read better as `PeripheryDeployConfig.s.sol` but i do see the benefit of having the files named alphabetically similarly

Reply:
sounds good, I updated the name to `PeripheryDeployConfig.s.sol`